### DISCUSSION TITLE
Adding optional arguments to `sep.Background` to return local background and rms levels before smoothing

### DISCUSSION THREAD

**Discussion by mapfiable** [sep-developers/sep#101] [Ideas]
In regards to better understanding what is happening "under the hood" it would be nice to have the option to take a look at the local background and rms maps before they are smoothed out. This may also lead to better parameter choices. 

Once you are able to inspect this level of the data, it might even make sense to have some control over the smoothing function, i.e. what kind and how drastically it is employed.

**Comment by kbarbary** [comment]
Do you mean the estimated value in each one of the `bw` x `bh` boxes, before the `fw` x `fh` filter is applied and before the interpolating spline is fit?

While there is not an option for different interpolation methods (the only option is a spline), one can control the degree of smoothing using the bw, bh, fw, fh parameters.

**Reply by mapfiable** [reply]
> Do you mean the estimated value in each one of the `bw` x `bh` boxes, before the `fw` x `fh` filter is applied and before the interpolating spline is fit?

Yes, that's what I mean. 

> While there is not an option for different interpolation methods (the only option is a spline), one can control the degree of smoothing using the bw, bh, fw, fh parameters.

That's true. Sorry, I forgot about that. Still, having different smoothing filters might be nice.
